### DISCUSSION TITLE
How to clear the cache of COG on SMARC IMX8

### DISCUSSION THREAD

**Discussion by yaoyaoTheCat** [Igalia/cog#448] [Q&A] [answered]
Hi all, any one knows how to clear the cache of COG on SMARC IMX8?

**Comment by philn** [comment, marked as answer]
Quoting https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html:

> $XDG_CACHE_HOME defines the base directory relative to which user-specific non-essential data files should be stored. If $XDG_CACHE_HOME is either not set or empty, a default equal to $HOME/.cache should be used.

**Reply by yaoyaoTheCat** [reply]
Thanks
